Parsers/CiscoUCS/CiscoUCS.txt

```diff
@@ -0,0 +1,70 @@
+// Title:           Cisco Unified Computing System (UCS)
+// Author:          Microsoft
+// Version:         1.0
+// Last Updated:    10/30/2020
+// Comment:         Inital Release
+//  
+// DESCRIPTION:
+// This parser takes raw Cisco UCS logs from a Syslog stream and parses the Audit, Fault and Event logs into a normalized schema.
+//
+// USAGE:
+// 1. Open Log Analytics/Azure Sentinel Logs blade. Copy the query below and paste into the Logs query window. 
+// 2. In the query window, on the second line of the query, enter the hostname(s) of your Cisco UCS device(s) and any other unique identifiers for the logstream. 
+//    For example: | where Computer in ("server1" , "server2")
+// 3. Click the Save button above the query. A pane will appear on the right, select "as Function" from the drop down. Enter a Function Name.
+//    It is recommended to name the Function Alias, as CiscoUCS
+// 4. Kusto Functions can typically take up to 15 minutes to activate. You can then use Function Alias for other queries.
+//
+// REFERENCES: 
+// Using functions in Azure monitor log queries: https://docs.microsoft.com/azure/azure-monitor/log-query/functions
+// 
+// LOG SAMPLES:
+// This parser assumes the raw log are formatted as follows:
+//
+// 1603399381.121    346 ::1 TCP_MISS/301 419 GET http://httpvshttps.com/ - HIER_DIRECT/45.33.7.16 text/html
+// 
+// 
+let LogHeader = Syslog
+| where Computer in ("UCSserver1", "UCSserver2")
```

Comment:
This should have a comment here that these names are examples and must be changed to your server names.

Reply:
comments have been added